TTHAnalysis/cfg/run_susyMultilepton_cfg.py

```diff
@@ -44,8 +44,14 @@
 # Lepton Skimming
```

Comment:
This and other modifications to configuration files are also in #142. It should not be the case... if left like this, one will need to be careful to merge correctly.